### PR TITLE
create UI ShareScreen

### PR DESCRIPTION
**Pull request by victorydst3**

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -28,6 +28,11 @@
     <style name="MyTheme.Fullscreen" parent="MyTheme">
         <item name="android:windowFullscreen">true</item>
     </style>
-
+    <style name="txt_home_screen">
+        <item name="android:layout_marginTop">@dimen/txt_home_screen_margin_top</item>
+        <item name="android:padding">@dimen/txt_home_screen_padding</item>
+        <item name="android:textSize">12sp</item>
```
Comment (tranminhlan) [inline app/src/main/res/values/styles.xml]: Please, use dimen .

File: app/src/main/res/layout/fragment_share.xml [hunk 2]
```diff
@@ -1,17 +1,101 @@
...
+            android:layout_width="75dp"
+            android:layout_height="45dp"
+            android:layout_alignParentRight="true"
+            android:layout_marginTop="10dp"
+            android:layout_toRightOf="@+id/imgCapture"
+            android:src="@drawable/selector_btn_recorded" />
 
-</RelativeLayout>
+        <TextView
+            android:layout_width="75dp"
```
Comment (tranminhlan) [inline app/src/main/res/layout/fragment_share.xml]: Please, use dimes

File: app/src/main/java/vn/asiantech/asiantechfamily/utils/CameraPreview.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+import android.app.Activity;
+import android.content.Context;
+import android.hardware.Camera;
+import android.view.Surface;
+import android.view.SurfaceHolder;
+import android.view.SurfaceView;
+
+import java.io.IOException;
+
+/**
```
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/utils/CameraPreview.java]: Please, remove all information not useful

File: app/src/main/java/vn/asiantech/asiantechfamily/fragments/ShareFragment.java [hunk 4]
```diff
@@ -16,28 +43,335 @@
  */
 
 @EFragment(R.layout.fragment_share)
-public class ShareFragment extends BaseFragment {
-    private View mView;
-
+public class ShareFragment extends BaseFragment implements SurfaceHolder.Callback, DialogConfirmPlayVideo.OnPlayVideo {
+    public final int SELECT_PHOTO = 1111;
```
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/fragments/ShareFragment.java]: i think you should use private in this case

Comment (victorydst3) [inline app/src/main/java/vn/asiantech/asiantechfamily/fragments/ShareFragment.java]: i already change code at new commit

File: app/src/main/java/vn/asiantech/asiantechfamily/activities/MainActivity.java [hunk 5]
```diff
@@ -24,6 +24,10 @@
 import vn.asiantech.asiantechfamily.fragments.Fragment;
 import vn.asiantech.asiantechfamily.views.HackyViewPager;
 
+
+/**
+ * @Create by mr.Tran on 10/08/2015
```
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/activities/MainActivity.java]: ERROR: Unknown tag 'Create'.

File: app/src/main/java/vn/asiantech/asiantechfamily/dialogs/DialogConfirmPlayVideo.java [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+                break;
+            case R.id.tv_dialog_cancel_log_out:
+                //call back cancle dialog
+                mOnPlayVideo.onCanCleDialog();
+                this.dismiss();
+                break;
+        }
+    }
+// call back play video after record
+    public interface OnPlayVideo {
```
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/dialogs/DialogConfirmPlayVideo.java]: ERROR: Missing a Javadoc comment.

File: app/src/main/java/vn/asiantech/asiantechfamily/eventbus/BusProvider.java [hunk 7]
```diff
@@ -2,13 +2,20 @@
 
 import com.squareup.otto.Bus;
 
+/**
+ * @Create by mr.Tran 10/02/2015
```
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/eventbus/BusProvider.java]: ERROR: Unknown tag 'Create'.

File: app/src/main/java/vn/asiantech/asiantechfamily/fragments/BaseContainerFragment.java [hunk 8]
```diff
@@ -40,18 +40,16 @@ public boolean popFragment() {
     public abstract android.support.v4.app.Fragment getChildFragment();
 
     @Override
-    public void onAttach(Activity _activity) {
-        super.onAttach(_activity);
-
-        activity = _activity;
+    public void onAttach(Activity activity) {
```
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/fragments/BaseContainerFragment.java]: ERROR: 'activity' hides a field.

File: app/src/main/java/vn/asiantech/asiantechfamily/fragments/BaseContainerFragment.java [hunk 9]
```diff
@@ -40,18 +40,16 @@ public boolean popFragment() {
...
+        super.onAttach(activity);
+        this.activity = activity;
         // Check if parent activity implements our callback interface
-        if (_activity != null) {
+        if (activity != null) {
             try {
                 Log.e("test", "pop fragment: " + getChildFragmentManager().getBackStackEntryCount());
-            } catch (ClassCastException e) {
+            } catch (ClassCastException ignored) {
```
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/fragments/BaseContainerFragment.java]: ERROR: Must have at least one statement.

File: app/src/main/java/vn/asiantech/asiantechfamily/activities/MainActivity.java [hunk 10]
```diff
@@ -32,6 +32,10 @@
 import vn.asiantech.asiantechfamily.fragments.BaseContainerFragment;
 import vn.asiantech.asiantechfamily.views.HackyViewPager;
 
+
+/**
+ * @Create by mr.Tran on 10/08/2015
```
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/activities/MainActivity.java]: ERROR: Unknown tag 'Create'.

File: app/src/main/java/vn/asiantech/asiantechfamily/dialogs/DialogConfirmPlayVideo.java (hunk 6 above)
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/dialogs/DialogConfirmPlayVideo.java]: ERROR: Missing a Javadoc comment.

File: app/src/main/java/vn/asiantech/asiantechfamily/eventbus/BusProvider.java (hunk 7 above)
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/eventbus/BusProvider.java]: ERROR: Unknown tag 'Create'.

File: app/src/main/java/vn/asiantech/asiantechfamily/fragments/BaseContainerFragment.java (hunk 8 above)
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/fragments/BaseContainerFragment.java]: ERROR: 'activity' hides a field.

File: app/src/main/java/vn/asiantech/asiantechfamily/fragments/BaseContainerFragment.java (hunk 9 above)
Comment (tranminhlan) [inline app/src/main/java/vn/asiantech/asiantechfamily/fragments/BaseContainerFragment.java]: ERROR: Must have at least one statement.

**Pull request merged**